### ISSUE TITLE
SQLite.SQLiteException: database is locked on startup

### ISSUE THREAD

**Issue by elliotbc** [doitandbedone/ispyagentdvr-docker#141]
**Affected Version**
Latest. repo digest: 
doitandbedone/ispyagentdvr@sha256:c3902be7ad569a3fdb37b1d090351d2266525575190933807553a4939c2e42a5

**Describe the bug**
When I start the container, I get an error about the SQLite database being locked

**Steps To Reproduce**
Note: port changed to 34789. Also changed in config XML file.
sudo docker run -it --net=host -p 8090:8090 -p 34789:3478/udp -p 50000-50010:50000-50010/udp \
-v /mnt/cams/ispyagentdvr/config/:/agent/Media/XML/ \
-v /mnt/cams/ispyagentdvr/media/:/agent/Media/WebServerRoot/Media/ \
-v /mnt/cams/ispyagentdvr/commands:/agent/Commands/ \
--name ispyagentdvr doitandbedone/ispyagentdvr
[ispy.log](https://github.com/doitandbedone/ispyagentdvr-docker/files/5827633/ispy.log)

**Expected behavior**
No error

**Reproduction rate**
Every time I try to start the container

**Screenshots**
N/A

**Host System Information (where ispy agent is running/hosted):**
Ubuntu 18.04

**Client System Information (device accessing the web portal, if applicable):**
N/A

**Additional Information**
See attached log file.

**Comment by doitandbedone** [comment]
Not sure how the TURN port can affect this but try removing the --net:host.
It also seems you kept the same container port 3478. So either change it to
the desired one or don't change the port in the xml file. Remember that
internally the container can have any port, it's the host's port that
matters as you may have conflicts with other services/containers.

Let me know if that works for you.

**Comment by doitandbedone** [comment]
@ispysoftware Have you seen this SQL error before?

**Comment by doitandbedone** [comment]
Are you installing this on a read-only or restricted partition or a system share?

There's some user that have the same issue on Docker for systems that either are shares which cause high traffic and lock the system or just a restricted/read-only partition:
https://www.reddit.com/r/docker/comments/cc7kb4/sonarrradarr_database_is_locked/
https://stackoverflow.com/questions/63221245/sqlite-database-locked-python-flask-on-ubuntu-docker-writing-to-azure-mounted

**Comment by ispysoftware** [comment]
there's a similar report here:
https://forum.duplicati.com/t/the-database-file-is-locked-error-even-when-the-file-is-newly-created/6893/13
with a comment:
"I experienced the same problem and the nolock option on the NFS mounted directory containing the database file solved the issue."

**Comment by elliotbc** [comment]
@doitandbedone 
Regarding the 3478 vs 34789, I changed the container port on the command line to 34789 and the behavior was the same. I wasn't thinking entirely clearly. I was only thinking that I needed to docker to forward port 34789 and that the web interface needed to reach out onto 34789. I didn't think much about the fact that the service on the container probably also changed ports with the XML change.

It's not a read-only partition, but it is samba mounted. It's a share on my freenas box that is exposed via samba.

I've attached my most recent log, and a file tree listing, and a cat of config.xml.

@ispysoftware 
Is there a similar option to try for samba?
[files_and_config.txt](https://github.com/doitandbedone/ispyagentdvr-docker/files/5833072/files_and_config.txt)
[ispy2.log](https://github.com/doitandbedone/ispyagentdvr-docker/files/5833073/ispy2.log)
